packages/static-wado-creator/lib/mkdicomwebConfig.js

```diff
@@ -238,6 +238,11 @@ const { mkdicomwebConfig } = ConfigPoint.register({
           reference: "rootDir",
         },
       },
+      {
+        key: "--hash-study-uid-path",
+        description: "Enable hashing of studyUID folder structure",
+        defaultValue: true,
```

Comment:
defaultValue: false please.  This is something we want to allow, but only big sites will use it